### PR TITLE
#165597934 Fix heroku deployment

### PR DESCRIPTION
**Pull request by mdn-network**
#### Description
Seeks to merge the Heroku integration to develop branch to enable other users to access and use the live application.

#### Type of change
- [ ] Bug fix (nonbreaking change which fixes an issue)
- [ ] New feature (nonbreaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### How Has This Been Tested?
Please describe the tests that you ran to verify your changes
- [ ] End to end
- [ ] Integration

#### Checklist:
- [ ] My code follows the style guide for this project
- [ ] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[165597934](https://www.pivotaltracker.com/story/show/165597934)

### CODE REVIEW

**Review by IssaIan — APPROVED** [review]
LGTM

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Review by Amoswachira — APPROVED** [review]
LGTM

**Review by a-braham — APPROVED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -1,22 +1,28 @@
 astroid==2.2.5
 atomicwrites==1.3.0
 attrs==19.1.0
+autopep8==1.4.4
 coverage==4.5.3
+dj-database-url==0.5.0
```
Comment (oluwagbenga-joloko) [inline requirements.txt]: is the package used in the project?

Comment (mdn-network) [reply requirements.txt]: For now it's not needed but cause i have not added any db url configutaions

Comment (oluwagbenga-joloko) [reply requirements.txt]: please remove it and push the updates.

**Pull request merged**